abel/tests/test_tools_center.py

```diff
@@ -5,39 +5,57 @@
 import os.path
 
 import numpy as np
-from numpy.testing import assert_allclose
+import numpy.testing as npt
```

Comment:
We can already access `assert_allclose` via `np.testing.assert_allclose`. The following is slightly more verbose, but I think it's more clear.